ote/src/cljs/ote/views/pre_notices/listing.cljs

```diff
@@ -24,11 +24,16 @@
 (defn pre-notices-table [e! pre-notices state]
   (let [notices (filter #(= state (::transit/pre-notice-state %)) pre-notices)]
     [:div.row
-     [table/table {:name->label (tr-key [:pre-notice-list-page :headers])
-                   :key-fn ::transit/id
-                   :no-rows-message (case state
-                                      :draft (tr [:pre-notice-list-page :no-pre-notices-for-operator])
-                                      :sent (tr [:pre-notice-list-page :no-pre-notices-sent]))}
+     [table/table (merge
+                    {:name->label (tr-key [:pre-notice-list-page :headers])
+                     :key-fn ::transit/id
+                     :no-rows-message (case state
+                                        :draft (tr [:pre-notice-list-page :no-pre-notices-for-operator])
+                                        :sent (tr [:pre-notice-list-page :no-pre-notices-sent]))}
+                    (when (= :sent state)
```

Comment:
Should drafts also open when clicking the row? Or should there be an "eye" action button to view.

May be confusing that two listings of the same things on the same page behave differently.